src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -96,6 +96,7 @@ private void testGeneralQuestionOperations() throws Exception {
 
         testEditQuestionLink();
         testEditQuestionAction();
+        testEditVisibilityAction();
```

Comment:
Just to check, is there anywhere else in this file where we edit the visibility of the question?